src/StackHCI/StackHCI.Autorest/Az.StackHCI.psd1

```diff
@@ -11,7 +11,7 @@
   DotNetFrameworkVersion = '4.7.2'
   RequiredAssemblies = './bin/Az.StackHCI.private.dll'
   FormatsToProcess = './Az.StackHCI.format.ps1xml'
-  FunctionsToExport = 'Add-AzStackHCIVMAttestation', 'Disable-AzStackHCIAttestation', 'Disable-AzStackHCIRemoteSupport', 'Enable-AzStackHCIAttestation', 'Enable-AzStackHCIRemoteSupport', 'Get-AzStackHciArcSetting', 'Get-AzStackHciCluster', 'Get-AzStackHciExtension', 'Get-AzStackHCILogsDirectory', 'Get-AzStackHCIRemoteSupportAccess', 'Get-AzStackHCIRemoteSupportSessionHistory', 'Get-AzStackHCIVMAttestation', 'Install-AzStackHCIRemoteSupport', 'Invoke-AzStackHciConsentAndInstallDefaultExtension', 'Invoke-AzStackHciExtendClusterSoftwareAssuranceBenefit', 'New-AzStackHciArcSetting', 'New-AzStackHciCluster', 'New-AzStackHciExtension', 'Register-AzStackHCI', 'Remove-AzStackHciArcSetting', 'Remove-AzStackHciCluster', 'Remove-AzStackHciExtension', 'Remove-AzStackHCIRemoteSupport', 'Remove-AzStackHCIVMAttestation', 'Set-AzStackHCI', 'Unregister-AzStackHCI', 'Update-AzStackHciCluster', '*'
+  FunctionsToExport = 'Add-AzStackHCIVMAttestation', 'Disable-AzStackHCIAttestation', 'Disable-AzStackHCIRemoteSupport', 'Enable-AzStackHCIAttestation', 'Enable-AzStackHCIRemoteSupport', 'Get-AzStackHciArcSetting', 'Get-AzStackHciCluster', 'Get-AzStackHciExtension', 'Get-AzStackHCILogsDirectory', 'Get-AzStackHCIRemoteSupportAccess', 'Get-AzStackHCIRemoteSupportSessionHistory', 'Get-AzStackHCIVMAttestation', 'Install-AzStackHCIRemoteSupport', 'Invoke-AzStackHciConsentAndInstallDefaultExtension', 'Invoke-AzStackHciExtendClusterSoftwareAssuranceBenefit', 'New-AzStackHciArcSetting', 'New-AzStackHciCluster', 'New-AzStackHciExtension', 'Register-AzStackHCI', 'Remove-AzStackHciArcSetting', 'Remove-AzStackHciCluster', 'Remove-AzStackHciExtension', 'Remove-AzStackHCIRemoteSupport', 'Remove-AzStackHCIVMAttestation', 'Set-AzStackHCI', 'Set-AzStackHciClusterRemoteSupport', 'Start-AzStackHciClusterLogCollection', 'Unregister-AzStackHCI', 'Update-AzStackHciCluster', '*'
```

Comment:
Did you test out Set-AzStackHciClusterRemoteSupport', 'Start-AzStackHciClusterLogCollection' what's the output?

Reply:
Yes i have added tests for both, you can check in the files